gm4_shapeless_portals/data/gm4_shapeless_portals/functions/x/build.mcfunction

```diff
@@ -0,0 +1,7 @@
+# fill in the portal frame
+# @s = portal frame marker
+# at @s
```

Comment:
`at @s` isn't very useful here. Perhaps change it to something like "at each closed space air block"

Reply:
This applies to `z` also